rotkehlchen/tests/api/test_eth2.py

```diff
@@ -1380,3 +1380,55 @@ def test_consolidated_validators_status(rotkehlchen_api_server: 'APIServer') ->
     assert result['entries'][0]['status'] == 'consolidated'
     assert result['entries'][0]['consolidated_into'] == 765881
     assert result['entries'][0]['validator_type'] == 'distributing'
+
+
+@pytest.mark.parametrize('ethereum_modules', [['eth2']])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_staking_performance_division_by_zero_protection(
```

Comment:
this can also just be a get_performance() unit test as you are just testing that function right?

But you also added some other things I don't know what they do.